CONTRIBUTING.md

```diff
@@ -127,11 +127,21 @@ To preview any changes to the documentation locally:
 
     # For contributors.
     uv pip install -r docs/requirements.txt
-   
+
     # Or, for members of the Astral org, which has access to MkDocs Insiders via sponsorship.
     uv pip install -r docs/requirements-insiders.txt
     ```
 
+1. Activate the virtual environment with:
+
+    ```shell
+    # On macOS and Linux.
+    source .venv/bin/activate
+
+    # On Windows.
+    .venv\Scripts\activate
+    ```
+
 1. Run the development server with:
```

Comment:
The bullet point numbers below this are now off

Reply:
In markdown, it doesn't matter which number you use for ordered lists, as using "1" for multiple consecutive items will still generate a numerically ordered list: https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Interesting, didn't know about that feature!